3_navigation.ipynb

```diff
@@ -261,25 +526,53 @@
     "</div>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "    <b>Advanced task</b>\n",
+    "    <p>Confirm what happens when we set a goal on the obstacle.</p>\n",
+    "    <p>Confirm what happens when we move a person in front of the robot while the robot is moving.</p>\n",
+    "    <p>Confirm what happens when we set a goal outside the map.</p>\n",
+    "</div>"
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# 自分で考えてみましょう。この下に入力できます。\n"
+    "# 自分で考えてみましょう。この下に入力できます。 Write your code here:\n"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "<div class=\"alert alert-block alert-info\">\n",
     "    <b>次の学習</b>\n",
     "    <p>今までの章では台車の制御を扱ってきました。次の章ではHSRのアームを制御してみましょう。</p>\n",
     "</div>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "    <b>What's next</b>\n",
+    "    <p>Now, we can use the mobile base very well. In the next notebook, we will learn how to use the arm of HSR.</p>\n",
```

Comment:
"    <p>Now, we can use the mobile base very well. In the next notebook, we will learn how to use the HSR's arm.</p>\n",